frontend/src/common/queries/wheresMyGene.ts

```diff
@@ -599,19 +591,13 @@ function useWMGQueryRequestBody(options = { includeAllFilterOptions: false }) {
   }, [data]);
 
   return useMemo(() => {
-    if (
-      !data ||
-      !selectedOrganismId ||
-      !selectedTissues.length ||
-      !selectedGenes.length
-    ) {
+    if (!data || !selectedOrganismId || !selectedTissues.length) {
       return null;
     }
-
     const gene_ontology_term_ids = selectedGenes.map((geneName) => {
       return organismGenesByName[geneName].id;
     });
-
+    if (!gene_ontology_term_ids.length) gene_ontology_term_ids.push(".");
```

Comment:
Passing "." as a gene param value here is a workaround that:

1. prevents the backend from throwing a "no gene specified" exception,
1. does not match on any gene ontology term ID, and
1. allows the backend to return all tissue names and cell types for the selected set of tissues.

I think we have two options here:

1. Leave this workaround as-is, add comments explaining the "." and also create a corresponding ticket to update the backend to handle the "no gene specified" case. The workaround will then be removed once the backend is updated.
1. Remove this workaround and update the backend to handle the "no gene specified case".

What are your thoughts, @seve?

Reply:
I think my preference would be to go with option 2 (update the backend to handle the "no gene specified" case) but am happy to go with the team's decision!

Reply:
If we want to get this unblocked asap, option 1 is best while we wait for folks to come back online.  I'd defer to @signechambers1

Reply:
this is not urgent so we can try for option 2, @atarashansky can you make the backend change or do we need to wait for madison to come back?

Reply:
@signechambers1 I can try to go for it, but there is some weirdness with how the backend deals with empty queries. That's why I opted for the frontend change. 

Personally, I think we should go with option 1. It's a bit of a hack but it won't require significant changes to the backend. If I were to do this in the backend, I'd do the same thing anyway - check if it's an empty gene list and then add a dummy gene to not break anything.

Reply:
@signechambers1 
I tried forcing the backend to accept no genes (by setting [this](https://github.com/chanzuckerberg/single-cell-data-portal/blob/ae3315869c4655b9e83db4c32bc435bb9a3dec9c/backend/wmg/data/query.py#L14) line), and it works - the only issue is that it takes forever to load because of how tiledb is handling the query with no genes selected. 

Simplest route forward would be to go with the frontend fix.

Reply:
ok, we can go with option 1. @atarashansky can you create a backend ticket for madison to take a look at to determine if we should eventually make the change there without negatively impacting performance?